src/System.Net.Http/tests/UnitTests/System.Net.Http.Unit.Tests.csproj

```diff
@@ -343,10 +341,11 @@
     <Compile Include="Headers\WarningHeaderValueTest.cs" />
     <Compile Include="HttpContentTest.cs" />
     <Compile Include="HttpRuleParserTest.cs" />
+    <Compile Include="Internal\MailBnfHelperTest.cs" />
     <Compile Include="MockContent.cs" />
   </ItemGroup>
   <ItemGroup>
     <None Include="project.json" />
   </ItemGroup>
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.targets))\dir.targets" />
-</Project>
+</Project>
```

Comment:
This change removed a newline from the end of the file.  The newline needs to be put back.